tests/unit/info.tcl

```diff
@@ -401,5 +401,41 @@ start_server {tags {"info" "external:skip"}} {
                 fail "pubsub clients did not clear"
             }
         }
+
+        test {clients: watching clients} {
+            assert_equal [s watching_clients] 0
+            set r2 [redis_client]
+            # count after watch
+            $r2 watch key
+            assert_equal [s watching_clients] 1
```

Comment:
i realize we don't have coverage for the watched key count (both in INFO and CLIENT INFO).
despite this being a test for INFO, i guess we can add some basic coverage here.

Reply:
:smile: now i'll ask if we really need to check all metrics on each and every step? or is it enough to have just one or two sanity checks in specific places?
this test is a little bit hard to follow..